src/nginx/habitat/config/chef_http_lb_common

```diff
@@ -16,7 +16,7 @@
     set_by_lua $data_collector_token 'return os.getenv("DATA_COLLECTOR_TOKEN")';
     {{~/if}}
 
-    access_log /dev/stdout opscode;
+    access_log {{cfg.access_log}} opscode;
```

Comment:
I built the pkg, exported to docker, started it via compose and noticed that this wasn't set.

```
/ # echo $HAB_CHEF_SERVER_NGINX
access_log = "/dev/tty"
/ # grep access_log /hab/svc/chef-server-nginx/config/chef_https_lb.conf
    access_log  opscode;
/ #
```

After glancing at the reset of the file, in this context, it should be `{{../cfg.access_log}}`